package.json

```diff
@@ -19,7 +19,13 @@
     "postcss": "8.4.23",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-icons": "^4.8.0",
     "tailwindcss": "3.3.2",
     "typescript": "5.0.4"
+  },
+  "devDependencies": {
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0"
```

Comment:
add those lines into devDependencies and run `npm install`
"@typescript-eslint/eslint-plugin": "^5.59.7",
"eslint-plugin-prettier": "^4.2.1",
after that `next lint` command should work